components/deployment-synchronizer/org.wso2.carbon.deployment.synchronizer/src/main/java/org/wso2/carbon/deployment/synchronizer/repository/CarbonRepositoryUtils.java

```diff
@@ -185,7 +185,7 @@ public static DeploymentSynchronizerConfiguration getDeploymentSyncConfiguration
 
             return config;
         }else{
-            return null;
```

Comment:
The "else" statement and probably the "if" are not properly code formatted. Even though these aren't your changes, when noticing this, its better to fix them right away.